submission/idr0000_KellerLab.txt

```diff
@@ -0,0 +1,97 @@
+# FILL IN AS MUCH INFORMATION AS YOU CAN.  HINTS HAVE BEEN PUT IN SOME FIELDS AFTER THE HASH # SYMBOL. REPLACE THE HINT WITH TEXT WHERE APPROPRIATE.				
+				
+# STUDY DESCRIPTION SECTION				
+"# Section with generic information about the study including title, description, publication details (if applicable) and contact details"				
+				
+Comment[IDR Study Accession]	# leave blank			
+Study Title	In toto imaging and reconstruction of post-implantation mouse development at the single-cell level			
+Study Type	# leave blank			
+Study Type Term Source REF	# leave blank			
+Study Type Term Accession	# leave blank			
+Study Description	"The mouse embryo has long been central to the study of mammalian development; however, elucidating the cell behaviors governing gastrulation and the formation of tissues and organs remains a fundamental challenge. A major obstacle is the lack of live imaging and image analysis technologies capable of systematically following cellular dynamics across the developing embryo. We developed a light-sheet microscope that adapts itself to the dramatic changes in size, shape and optical properties of the post-implantation mouse embryo and captures its development from gastrulation to early organogenesis at the cellular level. We furthermore developed a computational framework for reconstructing long-term cell tracks, cell divisions, dynamic fate maps and maps of tissue morphogenesis across the entire embryo. By jointly analyzing cellular dynamics in multiple embryos registered in space and time, we built a dynamic atlas of post-implantation mouse development that, together with our microscopy and computational methods, is provided as a resource."		 	
+Study Organism	"""Mus musculus"""			
+Study Organism Term Source REF	NCBITaxon 10090 			
+Study Organism Term Accession	# leave blank			
+Study Experiments Number	1			
+Study External URL	# if there is an existing web page related to the study enter it here			
+Study Public Release Date	"# enter a date of when the data can be made public, if known"			
+				
+# Study Publication				
+Study PubMed ID	# fill in if known			
+Study Publication Title	# fill in if known			
+Study Author List	"Katie McDole, Léo Guignard, Fernando Amat, Andrew Berger, Grégoire Malandain, Loïc A. Royer, Srinivas C. Turaga, Kristin Branson, Philipp J. Keller"			
+Study PMC ID	# fill in if known			
+Study DOI	# fill in if known			
+				
+# Study Contacts				
+Study Person Last Name	Keller	Kate	Leo	
+Study Person First Name	Philipp	McDole	Guignard	
+Study Person Email	kellerp@janelia.hhmi.org	mcdolek@janelia.hhmi.org	guignardl@janelia.hhmi.org	
+Study Person Address	19700 Helix Dr. Ashburn Va 20147 USA	19700 Helix Dr. Ashburn Va 20147 USA	19700 Helix Dr. Ashburn Va 20147 USA	
+Study Person Roles	submitter	submitter	submitter	
+				
+# Study License and Data DOI				
+Study License	# leave blank			
```

Comment:
This is something we might need to clarify with Kate et al i.e. are they fine with making the data `CC-BY 4.0` with copyright `McDole et al.` ? /cc @francesw